drivers/soundwire/intel.c

```diff
@@ -88,12 +105,10 @@
 #define SDW_ALH_STRMZCFG(x)		(0x000 + (0x4 * (x)))
 #define SDW_ALH_NUM_STREAMS		64
 
-#define SDW_ALH_STRMZCFG_DMAT_VAL	0x3
+#define SDW_ALH_STRMZCFG_DMAT_VAL	0xf
```

Comment:
As https://github.com/thesofproject/sof/pull/2226 is merged, I guess we don't need this change now

Reply:
will remove, thanks